README.md

```diff
@@ -35,12 +37,52 @@ jobs:
 This workflow definition scans your repository with Kubescape and publishes the results to Github.
 You can then see the results in the Pull Request that triggered the scan and the _Security → Code scanning_ tab.
 
+### Automatically Suggest Fixes
+
+To make Kubescape automatically suggest fixes to your pushes and pull requests, use the following workflow:
+
+```yaml
+name: Suggest autofixes with Kubescape
+on: [push,pull_request]
```

Comment:
```suggestion
on: [pull_request]
```
Because we are suggesting fixes on the branch, I think `push` should not be in our default examples